python/GafferSceneTest/CapsuleTest.py

```diff
@@ -135,5 +135,20 @@ def testInheritedAttributesAreNotBakedIntoCapsuleContents( self ) :
 			} )
 		)
 
+	def testCancellerNotStored( self ) :
+
+		sphere = GafferScene.Sphere()
+
+		hash = IECore.MurmurHash()
+		for path in ( "/", "/sphere" ) :
+			for method in ( "boundHash", "transformHash", "objectHash", "attributesHash" ) :
+				hash.append( getattr( sphere["out"], method )( path ) )
+
+		context = Gaffer.Context( Gaffer.Context(), IECore.Canceller() )
+		context["test"] = 1
+
+		capsule = GafferScene.Capsule( sphere["out"], "/", context, hash, sphere["out"].bound( "/" ) )
+		self.assertIsNone( capsule.context().canceller() )
```

Comment:
This test looks a bit weird - it goes to the effort of computing an accurate hash, and puts a "test" entry in the context so we can distinguish it from a default context ... but then never tests any of these things?

Reply:
> it goes to the effort of computing an accurate hash

The former I just copied from another test, not so much for testing purposes, but so that the test demonstrates that the Capsule maker is responsible for making a suitable hash. I think this is OK?

> and puts a "test" entry in the context so we can distinguish it from a default context

This was me completing botching things. I had intended to use `test` to check that the correct context got through (just not with the canceller). And of course that context should be scoped before the hash calls.

> doesn't need to prevent merging this if we're trying to get a release out.

I'm signing off now, and I agree this doesn't need to block the release. But I'll make a PR first thing tomorrow to improve the test...

Reply:
Thanks for catching that...